common/views/components/Icon/Icon.tsx

```diff
@@ -36,7 +36,7 @@ const Wrapper = styled.div.attrs({
   ${props =>
     props.color &&
     `
-  .icon__shape {
+  .icon__svg .icon__shape {
```

Comment:
why do we need to add this class to the selector?

Reply:
Agreed it's not great. It needed more specificity as it was being overwritten by its parent's `font-white`. I'm not sure why that changed, as it was always there, but it took precedence over it (I'm thinking the stylesheet insertion order has changed). It doesn't seem to happen elsewhere 🤞  

As discussed, I think there's another piece to do on `font-[color]` classes and stop them from influencing the icon's color (which should be either `currentColor` or the specified `color` prop ([which was missing and I added](https://github.com/wellcomecollection/wellcomecollection.org/pull/8408/files/f8d702904ae6fce696484c85e696fd04d331bda2#diff-0e3ccad0a512ac1c3682cc4b03820e0e8fb769b71b5f7f7c536337e0bfb177caR127) and still didn't overwrite `font-white`). 

* Funnily (hm) enough, I reran it and it looks like the insertion order changed again and when reverting my changes for this, the footer icons have the correct color again. Maybe it was a hot refresh issue (although I manually refreshed tons)? Happy to revert again and see...

Reply:
(Done for this file)